contracts/src/tests/fork_tests/test_ekubo.cairo

```diff
@@ -395,6 +420,11 @@ fn test_launch_meme_token0_price_above_1() {
         pool_key: pool_key
     );
 
+    // Change tx hash to avoid multicall detection
```

Comment:
what is it for here?

Reply:
I don't think you need this anywhere else than in the swap function

Reply:
@enitrat actually this is needed as we call `ekubo_launcher.withdraw_fees(id, recipient);` right after. If you remove it, test fail because of the multicall check

Reply:
ahhh because this triggers a memecoin transfer anyways, even if you only give the ETH back to the caller yeah